tests/integrations/realcluster/reboot_pd_test.go

```diff
@@ -4,14 +4,13 @@
 // you may not use this file except in compliance with the License.
 // You may obtain a copy of the License at
 //
-//     http://www.apache.org/licenses/LICENSE-2.0
+//	http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
why change it?

Reply:
After adding comments and uncommenting it, the IDE automatically formatted it. If it will have any impact, I can revert it

Reply:
got

Reply:
https://github.com/tikv/pd/assets/23556834/a973b044-7740-454a-ba8d-9838816bb825

Please let me keep it😭

Reply:
prefer to revert it